pkg/kopia/utils.go

```diff
@@ -56,6 +59,11 @@ const (
 	DataStoreGeneralContentCacheSizeMBKey = "dataStoreGeneralContentCacheSize"
 	// DataStoreGeneralMetadataCacheSizeMBKey is the key to pass metadata cache size for general command workloads
 	DataStoreGeneralMetadataCacheSizeMBKey = "dataStoreGeneralMetadataCacheSize"
+	// ServerUsernameFormat is used to construct server username for Kopia API Server Status Command
```

Comment:
```suggestion
	// ServerUsernameFormat is used to construct server username for Kopia Repository Server Status Command
```

Let's make sure we are using Kopia Repository Server and not Kopia API Server.